aim/db/models.py

```diff
@@ -161,6 +232,11 @@ def to_attr(self, session):
             attr = ('provided_contract_names' if c.provides
                     else 'consumed_contract_names')
             res_attr.setdefault(attr, []).append(c.name)
+        for d in res_attr.pop('openstack_vmm_domain_names', []):
```

Comment:
this should be pop('vmm_domains') and append only those ones which ahve type='OpenStack'